### DISCUSSION TITLE
Welcome to BthPS3 Discussions!

### DISCUSSION THREAD

**Discussion by nefarius** [nefarius/BthPS3#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChangoD123** [comment]
I installed this driver and now my Bluetooth adapter stopped working (error code 10) and I can't uninstall it without Bluetooth, any solution?
